package.json

```diff
@@ -45,7 +45,7 @@
     "main": "./client/dist/extension",
     "engines": {
         "node": ">=16.13.2",
-        "vscode": "^1.40.0"
+        "vscode": "^1.22.0"
```

Comment:
I don't think our LSP libraries support this old of a vscode version?

Reply:
Whoops. Thank you for the catch.